src/components/chips/chips.scss

```diff
@@ -55,7 +55,10 @@ $contact-chip-name-width: rem(12) !default;
   font-size: $chip-font-size;
   padding: $chip-wrap-padding;
   vertical-align: middle;
-  cursor: text;
+
+  &:not('.static-only') {
```

Comment:
actually readonly is a good idea. I have another PR that should use this style as well.